src/peripheral/mod.rs

```diff
@@ -100,13 +104,65 @@ pub struct Cpuid {
     pub isar: [RO<u32>; 5],
     reserved1: u32,
     /// Cache Level ID
+    #[cfg(armv7m)]
```

Comment:
Is that a problem? I can drop this cfg gate if you want, but it will never have done anything useful on ARMv6-M and `cortex-m` is still <v1.0 too.